rosbridge_server/scripts/rosbridge_websocket.py

```diff
@@ -115,7 +146,8 @@ def check_origin(self, origin):
     certfile = rospy.get_param('~certfile', None)
     keyfile = rospy.get_param('~keyfile', None)
     # if authentication should be used
-    authenticate = rospy.get_param('~authenticate', False)
+    authenticate = rospy.get_param('~authenticate')
```

Comment:
@Shokman is there any reason you changed removed the default value? I think this will break rosbridge if the parameter is absent. Can we set both auth parameters to have a False default?